### ISSUE TITLE
Viterbi backtracking parameter

### ISSUE THREAD

**Issue by balAdamT** [bmeaut/python_nlp_2018_spring#10]
In course_material/07_Tagging/07_Tagging_lecture.ipynb when updating the backtracking table, the following line is presented for the algorithm:
B[i,j]=argmaxkπ[k,j−1]⋅Tk,iB[i,j]=arg⁡maxkπ[k,j−1]⋅Tk,i 
Shouldn't this be also include ⋅Ei,j?

Other sources seem to include it and I can't think of any good explanation why omitting it would be reasonable.
Is this just a typo or was there some consideration behind this choice?

**Comment by juditacs** [comment]
You are correct, tt's indeed missing from the equation. The only difference between the two equations is max vs argmax.

**Comment by balAdamT** [comment]
Thanks
